tests/system/feature_store/test_spark_engine.py

```diff
@@ -1188,6 +1189,25 @@ def test_ingest_with_steps_drop_features(self):
         csv_path_storey = measurements.get_target_path(name="csv")
         read_and_assert(csv_path_spark, csv_path_storey)
 
+        measurements = fstore.FeatureSet(
+            "measurements_spark",
+            entities=[fstore.Entity(key)],
+            timestamp_key="timestamp",
+            engine="spark",
```

Comment:
The original issue wasn't for spark engine though. Need to add a test for the default engine too.

Reply:
fixed! in test_feature_store.py - > test_ingest_with_steps_drop_features method